tests/core/Resolver.spec.ts

```diff
@@ -287,13 +303,13 @@ describe('Resolver', () => {
       const actualNewSigningPublicKey2 = Document.getPublicKey(document, 'newKey2Updte1PostRec');
       expect(actualNewSigningPublicKey1).toBeDefined();
       expect(actualNewSigningPublicKey2).toBeDefined();
-      expect(document.public_keys.length).toEqual(2);
-      expect(actualNewSigningPublicKey1!.jwk).toEqual(newSigningPublicKey.jwk);
-      expect(actualNewSigningPublicKey2!.jwk).toEqual(newKey2ForUpdate1AfterRecovery.jwk);
-      expect(document.service_endpoints).toBeDefined();
-      expect(document.service_endpoints.length).toEqual(1);
-      expect(document.service_endpoints[0].endpoint).toBeDefined();
-      expect(document.service_endpoints[0].id).toEqual('newDummyHubUri2');
+      expect(document.publicKey.length).toEqual(2);
+      expect(actualNewSigningPublicKey1!.publicKeyJwk).toEqual(newSigningPublicKey.publicKeyJwk);
+      expect(actualNewSigningPublicKey2!.publicKeyJwk).toEqual(newKey2ForUpdate1AfterRecovery.publicKeyJwk);
+      expect(document.service).toBeDefined();
+      expect(document.service.length).toEqual(1);
+      expect(document.service[0].endpoint).toBeDefined();
```

Comment:
note: this is additional difference as compared to did-core. https://www.w3.org/TR/did-core/#service-endpoints
`service.endpoint` vs `service.serviceEndpoint`

Reply:
prefer `services.serviceEndpoint` here